scipy/spatial/_ckdtree.pyx

```diff
@@ -705,7 +698,8 @@ cdef class cKDTree:
 
             .. versionchanged:: 1.6.0
```

Comment:
Not sure if there's a policy on how to deal with `versionchanged`, but I'd update this to the last version where something changed.
```suggestion
            .. versionchanged:: 1.9.0
```